qa/test_template.sh

```diff
@@ -56,7 +56,7 @@ do
         # install packages
         inst=$($topdir/qa/setup_packages.py -i $i -u $pip_packages --cuda ${CUDA_VERSION})
         if [ -n "$inst" ]; then
-            pip install $inst -f /pip-packages
+            pip install --default-timeout=100 $inst -f /pip-packages
```

Comment:
I'm thinking, maybe it would be better to hit everything with 
`export PIP_DEFAULT_TIMEOUT=100` in the setup script?

But I see that the L3 jobs do not necessarily use it.